common/build/eslint-config-fluid/src/test/mockFiles/noReleaseTagMockClass.ts

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * @public
+ */
+class MockClass {
```

Comment:
We will need to add test cases for the other contexts we care about: at least interfaces and types.

It would also probably be a good idea to add a test case for class expressions as well, e.g.:
```typescript
const mockClass = class {
...
}
```

Reply:
I would recommend creating a separate test file for each of these concepts.